src/Routers/UsersRouter.js

```diff
@@ -150,6 +170,7 @@ export class UsersRouter extends ClassesRouter {
     this.route('POST', '/requestPasswordReset', () => {
       throw new Parse.Error(Parse.Error.COMMAND_UNAVAILABLE, 'This path is not implemented yet.');
     });
+    this.route('POST', '/requestPasswordReset', req => this.handleReset(req));
```

Comment:
Looks like this might be coming, but is currently missing, just putting the comment here to remind :P